test/performance/performance-tests.sh

```diff
@@ -31,6 +31,9 @@ function update_knative() {
   pushd .
   cd ${GOPATH}/src/knative.dev
   echo ">> Update istio"
+  # istio-pilot pod occasionally gets overloaded and dies, delete all pods from
+  # istio before reintalling it to get them freshly recreated
+  kubectl delete pods --all -n istio-system
```

Comment:
Should we delete deployments or even namespace instead?

Reply:
I feel deleting deployments and services is probably better, since if there is an update to istio, they will be created freshly after we apply the yamls.
I don't quite like deleting the namespace since I've seen quite a few times that the process gets stuck..